core/src/test/java/io/questdb/griffin/SqlCodeGeneratorTest.java

```diff
@@ -575,6 +575,47 @@ public void testCursorForLatestByOnSubQueryWithoutRandomAccessSupport() throws E
         );
     }
 
+    @Test
+    public void testJitIntervalFwdCursorBwdSwitch() throws Exception {
```

Comment:
nit: might be a good idea to move these new tests to `AsyncFilteredRecordCursorFactoryTest`. `SqlCodeGeneratorTest` contains way too much tests.

Reply:
done in https://github.com/questdb/questdb/pull/2393/commits/2f6f7d962344da7c7da10004272d1a2f5080b599